ferrocene/tools/symbol-report/src/main.rs

```diff
@@ -170,6 +176,12 @@ fn get_qualified_name(tcx: TyCtxt<'_>, def: LocalDefId) -> String {
     with_no_visible_paths!(with_resolve_crate_name!(with_no_trimmed_paths!(tcx.def_path_str(def))))
 }
 
+fn should_filter_out(qualified_name: &str) -> bool {
```

Comment:
let's comment why this exists. maybe something like "These functions exist only to allow code using `.subset` targets to compile, and will be removed once `#[ferrocene::certified]` is implemented"?